README.md

```diff
@@ -16,20 +16,20 @@ Zeebe provides visibility into and control over business processes that span mul
 * Export process data for monitoring and analysis
 * Engage with an active community
 
-[Learn more at zeebe.io](https://zeebe.io)
+[Learn more at camunda.com](https://camunda.com/products/cloud/workflow-engine/)
 
 ## Status
 
-Starting with Zeebe 0.20.0, the "developer preview" label was removed from Zeebe and the first production-ready version was released  .
+We are currently working on Zeebe 1.0.
 
-To learn more about what we're currently working on, please visit the [roadmap](https://zeebe.io/roadmap).
+To learn more about what we're currently working on, check the [GitHub issues](https://github.com/camunda-cloud/zeebe/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) and the [latest commits](https://github.com/camunda-cloud/zeebe/commits/develop).
```

Comment:
Maybe we could harness our issue `status` labels to make it clear to users what we are working on (`in progress`, `ready for review`) and what is coming we are working on soon `ready`.

Reply:
Something like that would help people understand what is happening / what is going to happen.

Reply:
https://github.com/zeebe-io/enhancements/blob/master/ZEP001-triage-process.md

Reply:
Might help to understand it better

Reply:
So this happens now? Once a week all issues are triaged?

Reply:
No, everyday new issues are triaged.